pages/handout/repo-setup.mdx

```diff
@@ -25,6 +26,35 @@ In your repository, go to *Settings -> Manage access* and invite your team membe
 as collaborators.
 
 
+## Setting up for automatic deployment
+
+We are taking advantage of GitHub Action and deploy design system when changes are pushed.
+But to make it working, you need a bit of setup.
+
+### 1. Define the `homepage` property
+You need to edit "homepage" property in ./design-system/package.json. It should be pointing to your repository.
```

Comment:
I think we don't need it for design system, it's only for create-react-app. It should deploy itself, every time the version it bump. Should I correct?

Reply:
I will fix